python/_tskitmodule.c

```diff
@@ -6303,6 +6670,46 @@ TreeSequence_get_site(TreeSequence *self, PyObject *args)
     return ret;
 }
 
+static PyObject *
+TreeSequence_get_metadata_schemas(TreeSequence *self) {
```

Comment:
Looking ahead a bit, maybe we should call this ``get_table_metadata_schemas``? It'll be a bit confusing to have both ``get_metadata_schema`` and ``get_metadata_schemas``.

Reply:
Good idea, fixed in https://github.com/tskit-dev/tskit/pull/542/commits/f89d21cfcc3e12219eb3fc0819e0e16a7370beac